src/components/ImageDetails/ImageDetails.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Input, Box, CloseButton } from '@chakra-ui/react';
+
+function ImageDetails({
+  index,
+  name,
+  preview,
+  // register,
+  defaultDescription,
+  removeImage,
+  removeDescription,
+  updateDescription,
+}) {
+  // let currentDescription;
+  // const changeDescription = e => {
+  //   updateDescription(index, e.target.value);
+  //   // updateDescription(index, e.target.value);
+  // };
+  const [currentDescription, setCurrentDescription] = useState();
+  // setFilesIntermediate(prev => {
+  //     prev[i].description = '';
+  //     return prev;
+  // })
+  return (
+    <Box>
+      <p>{name}</p>
+      <CloseButton
+        onClick={() => {
+          removeImage(index);
+          removeDescription(index);
+        }}
+      />
+      <img
+        key={preview}
+        alt="pic"
```

Comment:
Change the `alt` text to `Image of ${file_name}` for better accessibility.

Reply:
There was an eslint error when including the word image so I just set it to the file name